framework/src/binding/state/trie_db.rs

```diff
@@ -1,17 +1,25 @@
+use std::collections::HashMap;
 use std::path::Path;
 use std::sync::Arc;
 use std::time::Instant;
 
 use bytes::Bytes;
 use derive_more::{Display, From};
+use parking_lot::RwLock;
+use rand::{rngs::SmallRng, Rng, SeedableRng};
 use rocksdb::{Options, WriteBatch, DB};
 
 use common_apm::metrics::storage::{on_storage_get_state, on_storage_put_state};
 use protocol::{ProtocolError, ProtocolErrorKind, ProtocolResult};
 
+const RAND_SEED: u64 = 49999;
+const MAP_SIZE: usize = 2000;
```

Comment:
we can extract it to the node config

Reply:
The map size can be set in the config, however, the rand seed should hard code.

Reply:
I just talking about `map_size`

Reply:
ok